generics/constants/api-responses.js

```diff
@@ -121,5 +121,6 @@ module.exports = {
     "TEMPLATE_TASK_NOT_UPDATED": "Could not updated template task",
     "PROJECT_TEMPLATE_UPDATED" : "Successfully updated project template",
     "PROJECT_TEMPLATE_TASK_UPDATED" : "Successfully updated project template task",
-
+    "PROJECT_TEMPLATE_FETCH_ERROR" : "TemplateId or Link either one is required",
```

Comment:
@VISHNUDAS-tunerlabs change "TEMPLATE_ID_OR_LINK_REQUIRED" : "TemplateId or Link either one is required",